docs/source/tutorial.rst

```diff
@@ -206,15 +229,15 @@ Let’s see what type of value we get for different integrators.
 
 .. parsed-literal::
 
-    **Output:** Results: 21.83991813659668
-            Abs. Error: 9.06180573e+00
-            Rel. Error: 7.09166229e-01
+    **Output:** RuntimeError: result type ComplexFloat can't be cast to the desired output type Float
```

Comment:
??? This is very awkward.

Reply:
Would you rather hide it?

Reply:
yes don't run vegas for it if incompatible